### DISCUSSION TITLE
Node Notes Section

### DISCUSSION THREAD

**Discussion by NerdsCorp** [pelican-dev/panel#448] [Ideas]
Would it be possible to get a section for note on a node. The purpose would be to add notes regarding the node itself like, info on how long the device has been in service, I could add notes regarding where it is in my home lab, or any info for the Node itself like service date and things of that nature.

**Comment by rmartinoscar** [comment]
Can be closed as discussed in [PR #450](https://github.com/pelican-dev/panel/pull/450)

**Reply by Poseidon281** [reply]
Where is it discussed that this should be closed? That is what your opinion was?

**Reply by rmartinoscar** [reply]
OP's comment https://github.com/pelican-dev/panel/pull/450#issuecomment-2198794203
Also Lance closing the PR made me feel that this discussion was out of date i might be wrong tho

**Reply by Poseidon281** [reply]
I think that it was a good idea to have a notes section and Lance closed it, because it was not discussed in a issue first
